lib/group-id.js

```diff
@@ -42,7 +42,8 @@ function fromMsgKey (msg, msgKey) {
 }
 
 function fromGroupKey (msg, groupKey) {
-  const { author, previous, content } = msg.value
+  const { author, previous } = msg.value
+  const content = (msg.meta && msg.meta.originalContent) || msg.value.content
```

Comment:
```suggestion
  const content = msg?.meta?.originalContent || msg.value.content
```

Reply:
i've avoided `?.` to make it easier to potentially support node 12 again in the future, but maybe we've dropped the hopes for that?